.github/workflows/My-ci-pipeline.yml

```diff
@@ -0,0 +1,34 @@
+name: Continuous Integration
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+      - '**.lock'
+      - '.github\**'
+      - '.idea\**'     
+      - '.pytest_cache\**'
+      - '.venv\**'
+      - '.vscode\**'
+
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '**.lock'
+      - '.github\**'
+      - '.idea\**'     
+      - '.pytest_cache\**'
+      - '.venv\**'
```

Comment:
These two paths should be able to be skipped, because the cache & venv aren't checked in with Git anyway so they shouldn't be updating on your repository

Reply:
removed